src/main/java/net/fabricmc/fabric/mixin/client/model/MixinBakedModelHelper.java

```diff
@@ -0,0 +1,14 @@
+package net.fabricmc.fabric.mixin.client.model;
+
+import net.minecraft.util.math.Direction;
+
+class MixinBakedModelHelper {
+    /**
+     * This is here for use by {@link MixinBakedModel} which cannot define
+     * static fields without Mixin complaining.  The implementation there
```

Comment:
It can, as long as they're private.

Reply:
If I could make MixinBakedModel abstract that would certainly work, but believe it has to be an interface because BakedModel is. 

Is there a sneaky way to declare private members in an interface?

Reply:
Yes! In later versions of Java :D.

Reply:
Whoa, should I be targeting J9?  Totally missed that if so.  Would be nice.

So used to that other one...  haven't even really looked at J9 much.

Reply:
I wish! That was TOTALLY NOT what I tried to imply. Nope. We have to support J8.

Reply:
Ah, well.  So, leave it for now?  Accept the potential allocation hit?  Is there a less bad place to stash the ugliness?

Reply:
Yes.

api/ is public-facing classes and interfaces, impl/ is our internal details, mixin/ is MIXINS ONLY. As such, such a helper class should be thrown in impl/, with just a public static final field...